raft.go

```diff
@@ -488,7 +607,7 @@ func (r *Raft) Stats() map[string]string {
 		"fsm_pending":         toString(uint64(len(r.fsmCommitCh))),
 		"last_snapshot_index": toString(lastSnapIndex),
 		"last_snapshot_term":  toString(lastSnapTerm),
-		"num_peers":           toString(uint64(len(r.peers))),
+		"num_servers":         toString(uint64(len(r.configurations.latest.Servers))),
```

Comment:
Elsewhere we've taken to including the list of servers along with the number of peers.  e.g.  https://github.com/hashicorp/nomad/blob/c6b7bb66f309b7fc3ef325b158115aa046dcbf30/nomad/server.go#L905-L906